usecases/modules/module_config_init_and_validate_test.go

```diff
@@ -216,6 +216,57 @@ func TestValidateClass(t *testing.T) {
 	})
 }
 
+func TestSetSinglePropertyDefaults(t *testing.T) {
+	t.Run("add property after creating the class", func(t *testing.T) {
```

Comment:
I think we do not need the t.run here. That is mainly used when you have multiple sub_tests that should run independently.

Reply:
Makes sense. Fixed.